source/vibe/core/core.d

```diff
@@ -851,6 +851,21 @@ void switchToTask(Task t, TaskSwitchPriority priority)
 	wait time, in contrast to $(D core.thread.Thread.sleep), which shouldn't be
 	used in vibe.d applications.
 
+    Repeated_sleep:
+      As this method creates a new `Timer` every time, it is not recommended to
+      use it in a loop. For this purpose, prefer to instantiate a single `Timer`
```

Comment:
I'd slightly reword this to something like "tight loop" or "called very frequently", as the creation overhead will usually only matter once the timeout enters millisecond territory. Might make sense to change this to maintain a small stack of reusable sleep timers per thread to avoid this issue.